tools/generators/migrate-converged-pkg/index.spec.ts

```diff
@@ -45,6 +45,12 @@ describe('migrate-converged-pkg generator', () => {
   let tree: Tree;
   const options = { name: '@proj/react-dummy' } as const;
 
+  const getScopedPkgName = (pkgName: string) => {
```

Comment:
**suggestion:** we can make it a `function` and move it to the bottom (function declarations are hoisted )